src/naive/date.rs

```diff
@@ -1389,6 +1412,30 @@ impl NaiveDate {
         NaiveWeek { date: *self, start }
     }
 
+    // This duplicates `Datelike::year()`, because trait methods can't be const yet.
+    #[inline]
+    const fn _year(&self) -> i32 {
```

Comment:
What happens if we skip the `_` prefix here?

Reply:
To my surprise nothing blows up :smile:

Reply:
Rustdoc was not happy about it, but that is fixable.
https://github.com/chronotope/chrono/actions/runs/5694596475/job/15436073978?pr=1205